deploy/aws/conf/cloudformation_template.json

```diff
@@ -304,6 +304,7 @@
                   "Effect": "Allow",
                   "Action": [
                     "sqs:DeleteMessage",
+                    "sqs:GetQueueAttributes",
```

Comment:
We are calling that API here: https://github.com/blox/blox/blob/dev/cluster-state-service/handler/event/sqs_consumer.go#L110